.ci/install_crio.sh

```diff
@@ -126,7 +126,7 @@ make clean
 make BUILDTAGS='seccomp selinux exclude_graphdriver_btrfs exclude_graphdriver_devicemapper libdm_no_deferred_remove'
 make test-binaries
 sudo -E PATH=$PATH sh -c "make install"
-sudo -E PATH=$PATH sh -c 'crio -d "" config > crio.conf'
+sudo -E PATH=$PATH sh -c 'crio -d "" --cgroup-manager "systemd" config > crio.conf'
```

Comment:
See previous comment, it seems that here we can use `CONF_OVERRIDES` and just call `make install`.

Reply:
Check the answer for the previous comment. :-)